src/styles/table.styl

```diff
@@ -48,3 +48,20 @@
         .set-table-date
         .set-table-actions
             display none
+
+.row
+    height 64px
+
+.browser
+    padding 24px
+
+.main
+    display flex
+    flex-direction column
+
+.title
+    font-size 1rem
+    color var(--black)
```

Comment:
You should not use color based variable. You should use semantics ones. 

cc @JF-Cozy

Reply:
I will use : `--primaryTextColor`